src/pages/Main/MainSlides/MainSlides.scss

```
@@ -0,0 +1,29 @@
+//scss
+.mainSlides {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  overflow-y: hidden;
+
+  .container {
+    width: 100%;
+    height: 100vh;
+    background-color: rgb(177, 177, 177);
+    overflow: hidden;
+  }
+
+  button {
+    position: absolute;
+    top: 50vh;
```

정중앙으로 정렬하는 것이 목적이라면, 정중앙에 정렬되지는 않을 것 같네요!
```suggestion
    top: 0;
    bottom: 0;
```
```suggestion
    top: 50vh;
    transform: translateY(-50%);
```